testsuite/drivers/helpers.py

```diff
@@ -89,6 +93,8 @@ def distribution():
 
         return 'DISTRO_UNKNOWN'
 
+    elif on_macos():
```

Comment:
This will have to be changed according to the comments about MacOS vs Homebew distrib.

Reply:
I think what we might want to do is have a CI run that use Homebrew (`eval $(brew shellenv)`) and one that doesn't. @mosteo  ?

Reply:
Sounds right to me, just as we test with no package manager on linux and with the supported ones too.

Reply:
I could certainly set this up.

I did think about checking whether `brew` is on the `PATH`, but users might want to use a different package manager, & if they had both this could lead to confusion; checking for `HOMEBREW_PREFIX` would mean they were OK with it.

Would have to change `testsuite/drivers/helpers.py` to make the same choice: e.g. like [this](https://github.com/simonjwright/alire/commit/50f4fe9ed5cd6aa037ea9a6558a984bde9bb2b46) and [this](https://github.com/simonjwright/alire/commit/913e5573e3d47557d2e750fe2e6e71f2202a3eec), successful run [here](https://github.com/simonjwright/alire/actions/runs/3077781843)